index.html

```diff
@@ -80,5 +80,192 @@ <h2 id="h2"><p>I'm a Software Developer.</p></h2>
+            </action>
+            <action_btn id="action_btn" class="orange_btns"><p id="action_btn_text">See Project</p>
+            </action_btn>
+
+            <li id="card_data">
+              <card_title id="card_title"><p>Profesional Art Printing Data</p></card_title>
+              <card_secondary id="card_secondary"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p></card_secondary>
+              <ul id="card_tags">
+                <li class="card_btns"><card_btns_font class="card_btns_font">html</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">bootstrap</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">Ruby</card_btns_font></li>
+                  </ul>
+
+              </card_tags>
+            </li>
+          </card_works>
+
+          <card_works id="card_works">
+            <action id="action">
+              <img src="./assets/Mask Group.png" alt="">
+            </action>
+            <action_btn id="action_btn" class="orange_btns"><p id="action_btn_text">See Project</p>
+            </action_btn>
+
+            <li id="card_data">
+              <card_title id="card_title"><p>Profesional Art Printing Data</p></card_title>
+              <card_secondary id="card_secondary"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p></card_secondary>
+              <ul id="card_tags">
+                <li class="card_btns"><card_btns_font class="card_btns_font">html</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">bootstrap</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">Ruby</card_btns_font></li>
+                  </ul>
+
+              </card_tags>
+            </li>
+          </card_works>
+        </ul>
+    </frame44>
+
+    <about_me id="about_me">
+      <me_primary_text><p id="me_primary_text">About me</p></me_primary_text>
+      <me_secondary_text><p id="me_secondary_text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></me_secondary_text>
+      <a href="https://github.com/glitcher255/micro-porfolio"><me_btn class="orange_btns" id="me_btn"><p id="me_btn_text">Get My Resume</p></me_btn></a>
+      <me_art id="me_art"></me_art>
+      <me_rectangle id="me_rectangle"><img src="./assets/Rectangle 73.png" alt=""></me_rectangle>>
+      <me_vector id="me_vector"><img src="./assets/Vector 4.png" alt=""></me_vector>
+    </about_me>
+
+  <skill id="skill">
+    <ul id="frame30">
+      <frame30_rectangle id="frame30_rectangle"></frame30_rectangle><p id="language_text">language</p></frame30_rectangle>
```

Comment:
Kindly suggest that when you use UL/LI listings, just put LI listings, remove other tags

Reply:
This breaks formatting and is not a part of this project, this is extra code for next week's project, also can you please clarify a bit more what you mean by other tags?

Reply:
Only LI inside UL lists.
https://www.w3schools.com/html/html_lists.asp

Reply:
This simply breaks my code, so I just removed it as its not required for now.